genesyscloud/architect_flow/resource_genesyscloud_architect_flow_schema.go

```diff
@@ -0,0 +1,87 @@
+package architect_flow
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"terraform-provider-genesyscloud/genesyscloud/validators"
+
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	registrar "terraform-provider-genesyscloud/genesyscloud/resource_register"
+)
+
+const (
+	resourceName = "genesyscloud_flow"
+)
+
+// SetRegistrar registers all resources, data sources and exporters in the package
+func SetRegistrar(l registrar.Registrar) {
+	l.RegisterDataSource(resourceName, DataSourceArchitectFlow())
+	l.RegisterResource(resourceName, ResourceArchitectFlow())
+	l.RegisterExporter(resourceName, ArchitectFlowExporter())
+}
+
+func ArchitectFlowExporter() *resourceExporter.ResourceExporter {
+	return &resourceExporter.ResourceExporter{
+		GetResourcesFunc: provider.GetAllWithPooledClient(getAllFlows),
+		RefAttrs:         map[string]*resourceExporter.RefAttrSettings{},
+		UnResolvableAttributes: map[string]*schema.Schema{
+			"filepath": ResourceArchitectFlow().Schema["filepath"],
+		},
+		CustomFlowResolver: map[string]*resourceExporter.CustomFlowResolver{
+			"file_content_hash": {ResolverFunc: resourceExporter.FileContentHashResolver},
+		},
+	}
+}
+
+func ResourceArchitectFlow() *schema.Resource {
+	return &schema.Resource{
+		Description: `Genesys Cloud Flow`,
+
+		CreateContext: provider.CreateWithPooledClient(createFlow),
+		UpdateContext: provider.UpdateWithPooledClient(updateFlow),
+		ReadContext:   provider.ReadWithPooledClient(readFlow),
+		DeleteContext: provider.DeleteWithPooledClient(deleteFlow),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
+			"filepath": {
```

Comment:
Did forceNew used to be on this attribute? The description implies it should be forceNew but I checked the schema on main and it does not have forceNew either.

Reply:
Not going to do right now as I have not touched the existing schema.